development/cpp/configuring_an_ide/visual_studio_code.rst

```diff
@@ -140,7 +143,7 @@ To run and debug the project you need to create a new configuration in the ``lau
       "name": "Launch Project",
       "type": "cppvsdbg",
       "request": "launch",
-      "program": "${workspaceFolder}/bin/godot.windows.tools.64.exe",
+      "program": "${workspaceFolder}/bin/godot.windows.editor.dev.x86_64.exe",
       // Change the arguments below for the project you want to test with.
       // To run the project instead of editing it, remove the "--editor" argument.
       "args": [ "--editor", "--path", "path-to-your-godot-project-folder" ],
```

Comment:
There is one more screenshot below which needs to be changed :):
https://docs.godotengine.org/en/latest/_images/vscode_2_launch.json.png

Reply:
updated, thanks for the hint